pkg/schedule/diagnostic/diagnostic_manager.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 TiKV Project Authors.
```

Comment:
```suggestion
// Copyright 2023 TiKV Project Authors.
```

Reply:
I think it's just a movement, maybe we don't need to change it?